src/Mod/Draft/drafttaskpanels/task_base.py

```diff
@@ -0,0 +1,362 @@
+        self.use_link = utils.get_param("Draft_array_Link", True)
+
+        self.form.checkbox_fuse.setChecked(self.fuse)
+        self.form.checkbox_link.setChecked(self.use_link)
+        # -------------------------------------------------------------------
+
+        # Some objects need to be selected before we can execute the function.
+        self.selection = None
+
+        # This is used to test the input of the internal function.
+        # It should be changed to True before we can execute the function.
+        self.valid_input = False
+
+        self.set_widget_callbacks()
+
+        self.tr_true = QT_TRANSLATE_NOOP("Draft", "True")
+        self.tr_false = QT_TRANSLATE_NOOP("Draft", "False")
+
+        # The mask is not used at the moment, but could be used in the future
+        # by a callback to restrict the coordinates of the pointer.
+        self.mask = ""
+
+    def set_widget_callbacks(self):
+        """Set up the callbacks (slots) for the widget signals."""
+        # New style for Qt5
+        self.form.button_axis.clicked.connect(self.select_axis)
+        self.form.button_reset.clicked.connect(self.reset_point)
+
+        # When the checkbox changes, change the internal value
+        self.form.checkbox_fuse.stateChanged.connect(self.set_fuse)
+        self.form.checkbox_link.stateChanged.connect(self.set_link)
+
+        # Old style for Qt4, avoid!
+        # QtCore.QObject.connect(self.form.button_reset,
+        #                        QtCore.SIGNAL("clicked()"),
+        #                        self.reset_point)
+        # QtCore.QObject.connect(self.form.checkbox_fuse,
+        #                        QtCore.SIGNAL("stateChanged(int)"),
+        #                        self.set_fuse)
+        # QtCore.QObject.connect(self.form.checkbox_link,
+        #                        QtCore.SIGNAL("stateChanged(int)"),
+        #                        self.set_link)
+
+    def get_center(self):
+        """Get the value of the center from the widgets."""
+        c_x_str = self.form.input_c_x.text()
+        c_y_str = self.form.input_c_y.text()
+        c_z_str = self.form.input_c_z.text()
+        center = App.Vector(U.Quantity(c_x_str).Value,
+                            U.Quantity(c_y_str).Value,
+                            U.Quantity(c_z_str).Value)
+        return center
+
+    def select_axis(self):
+        """Execute as callback when the axis button is clicked.
+
+        The axis selection process and button has the three stages
+        select (triggers selection), abort (aborts selection),
+        discard (set after axis is selected successfully and discards
+        the selected and saved axis)
+        """
+        if self.form.button_axis.text() == "select":
+            self.enable_axis_selection()
+            return
+        elif self.form.button_axis.text() == "abort"\
+                or self.form.button_axis.text() == "discard":
+            self.disable_axis_selection()
```

Comment:
Instead of using backslash continuation, you can use the parentheses. This is more Pythonic.
```py
elif (self.form.button_axis.text() == "abort"
      or self.form.button_axis.text() == "discard"):
self.disable_axis_selection()
```